### PR TITLE
Debug install block and rework on grid small size screen

### PR DESCRIPTION
**Pull request by 0xMazout**

### CODE REVIEW

**Comment by 0xMazout** [comment]
Detected a bug  after my pr was merged , on the install block this is to do a quickfix

File: web/home/src/pages/Home/home.tsx [hunk 1]
```diff
@@ -136,15 +134,11 @@ function Home() {
...
       });
   };
-
-  const defineGridStyle = (length: number) => {
-    let styles = gridStyle;
-    return (styles += length <= 3 ? ' grid-cols-3 ' : setResponsiveGrid);
+  const defineGridStyle = () => {
+    const setResponsiveGrid = `max-sm:grid-cols-1 sm:grid-cols-2 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4`;
+    const gridStyle = 'grid grid-flow-row mx-auto mt-3 gap-4';
+    return gridStyle + ' ' + setResponsiveGrid;
```
Comment (peterjah) [inline web/home/src/pages/Home/home.tsx:140]: why a function ? it returns a constant. Just use a constant

File: web/plugin-manager/src/pages/Manager.tsx [hunk 2]
```diff
@@ -105,35 +105,29 @@ function Manager() {
...
     };
 
-    const defineGridStyle = (length: number) => {
-        const numberOfCols = 3;
-        let styles = gridStyle;
-        return (styles += length <= numberOfCols ? " grid-cols-3 " : setResponsiveGrid);
+    const defineGridStyle = () => {
+        const setResponsiveGrid = `max-sm:grid-cols-1 sm:grid-cols-2 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4`;
+        const gridStyle = "grid grid-flow-row mx-auto mt-3 gap-4";
+        return gridStyle + " " + setResponsiveGrid;
```
Comment (peterjah) [inline web/plugin-manager/src/pages/Manager.tsx:111]: same here. no function needed

File: web/plugin-manager/src/components/LabelButton.tsx [hunk 3]
```diff
@@ -20,6 +22,10 @@ const LabelButton = (props: Props) => {
     function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
         event.preventDefault();
         props.callbackToParent(value);
+        setIsPending(true);
+        setTimeout(() => {
+            setIsPending(false);
+        }, 4000);
```
Comment (peterjah) [inline web/plugin-manager/src/components/LabelButton.tsx:28]: so the isPending state will just stay 4 seconds an disappear ?? :/ is it really what we want

**Review by peterjah — CHANGES_REQUESTED** [review]

File: web/home/src/pages/Home/home.tsx (hunk 1 above)
Comment (0xMazout) [reply web/home/src/pages/Home/home.tsx:140]: you're right it was not like this at the beginning !

File: web/plugin-manager/src/components/LabelButton.tsx (hunk 3 above)
Comment (0xMazout) [reply web/plugin-manager/src/components/LabelButton.tsx:28]: it was a dirty temporary choice but i can spend more times on this. We have to do prop drilling on 4levels for this

(it was out of the scope at the first time)

**Review by peterjah — APPROVED** [review]

**Review by Thykof — APPROVED** [review]

**Pull request merged**